search-app/docs/api.md

```diff
@@ -2473,6 +2474,48 @@ __Example Response__
 }
 ```
 
+### <a name="deleted-collections"></a> Find collections that have been deleted
+
+To support metadata harvesting, a harvesting client can search CMR for collections that are deleted within a time period. The only search parameter supported is `revision_date` and its format is the same as the `revision_date` parameter in regular collection search. The only supported result format is xml references. The response is the references to the highest non-tombstone collection revisions of the collections that are deleted within the given revision date period. e.g.
```

Comment:
As I have commented on the previous comment, we have not supported any formats other than xml for all revisions search. Most of the consideration is probably performance related. It could become really expensive to support more heavyweight formats with all revisions search. For now, we don't have a use case to have to do this. We can talk about it if you really want to look into it.